apps/studio/pages/support/new.tsx

```diff
@@ -82,9 +86,16 @@ const SupportPage = () => {
             ].join(' ')}
           >
             {sentCategory !== undefined ? (
-              <Success sentCategory={sentCategory} />
```

Comment:
Is this even something you changed? Seems kinda unrelated and might just wrongfully show up here?

Reply:
No, that's not my changes.